ramlfications/parser/main.py

```diff
@@ -372,12 +372,70 @@ def create_resources(node, resources, root, parent):
             else:
                 child = _create_resource_node(name=k, raw_data=v, method=None,
                                               parent=parent, root=root)
-                resources.append(child)
+                if "type" in v:
+                    # There may be some more methods defined on the resource
+                    # type this one inherits from
+                    supertype = v["type"]
```

Comment:
I prefer the `_get(v, "type")` approach here since it's used elsewhere. e.g.:

```python
if _get(v, "type"):
    resources = _create_supertype_resources(root, k, parent, _get(v, "type"), resources)
```